scripts/base/frameworks/logging/writers/ascii.zeek

```diff
@@ -54,6 +54,11 @@ export {
 	## This option is also available as a per-filter ``$config`` option.
 	const gzip_file_extension = "gz" &redef;
 
+  ## Default logs to current directory
+	## Can be redefined to send files into logging directory
+	##
+	const logdir = "." &redef;
```

Comment:
I think I would prefer the default to be "". That way the default behavior is the same as before and the code-path is not taken in the ascii writer.

Reply:
No problem, changed